plugin.yml

```diff
@@ -1,9 +1,47 @@
-name: Pact cli
-description: Pact cli utilities
+name: Pact CLI
+description: Pact CLI helper
 author: https://tapendium.com
 requirements:
   - curl
   - jq
   - docker
   - buildkite-agent
-configuration: ~
+configuration:
+  properties:
+    debug:
+      type: string
+      description: Enable debug mode
+    pacticipant:
+      type: string
+      description: Name of consumer/provider e.g. service
+    action:
+      type: string
+      enum:
+        - pr
+        - merge
+      description: The type of pipeline
```

Comment:
does it have a default or we will need to specify one?

Reply:
no default but I've created this ticket --> https://benieworldwide.atlassian.net/browse/LIBS-232 to work it out based on the current pipeline name so this option will be an override instead

Reply:
Noice!